CDTDatastoreTests/OHHTTPStubsHelper.h

```diff
@@ -0,0 +1,26 @@
+//
+//  OHHTTPStubsHelper.h
+//
+//  Copyright © 2017 IBMCloudant. All rights reserved.
```

Comment:
Haha yes, I guess I got distracted when I was updating that and helpfully copied it to the other file as well.
Fixed in 37f94a91ef9b9dc0136f4c84411d18e28baa2805